testsuite/classlibrary/TestSoundFile.sc

```diff
@@ -0,0 +1,63 @@
+TestSoundFile : UnitTest {
+
+	var server, soundFile, path, info;
+
+	setUp {
+
+		path = Platform.resourceDir +/+ "sounds/a11wlk01.wav";
+		info = IdentityDictionary[
+			(\numFrames -> 188893),
+			(\numChannels -> 1),
+			(\path -> path),
+			(\sampleRate -> 44100),
+			(\sampleFormat -> "int16"),
+			(\headerFormat -> "WAV")
+		];
+
+		server = Server(this.class.name);
+		this.bootServer(server);
+		
+		soundFile = SoundFile(path);
+		server.sync;
+		
+	}
+
+	tearDown {
+
+		if (soundFile.isOpen) { soundFile.close };
+		server.quit.remove;
+
+	}
+
+	test_isOpen {
+
+		soundFile.openRead;
+		server.sync;
+
```

Comment:
A `server.sync` makes no sense here, because the server isn't involved in `openRead`.

Reply:
Right. But we need to wait a little bit, no? I can change this to `0.001.wait` or similar.

Reply:
No, you don't. SoundFile openRead is 100% synchronous.